apps/AEPSampleApp/extensions/EdgeIdentity.js

```diff
@@ -39,6 +43,90 @@ function getExperienceCloudId() {
   AEPIdentity.getExperienceCloudId().then(experienceCloudId => console.log("AdobeExperienceSDK: Experience Cloud Id = " + experienceCloudId));
 }
 
+function getIdentities() {
+  AEPIdentity.getIdentities().then(identities => console.log("AdobeExperienceSDK: Get AEPIdentity Maps = " + JSON.stringify(identities)));
+}
+
+function updateIdentities() {
+
```

Comment:
Let's maintain a list of identities. We insert by incrementing index and append it to all the items. and when we remove we just remove the last inserted or first inserted whatever is the preference. Or we can keep textbox to pass index for the item to be removed.

Reply:
I will update sample app in another PR